### ISSUE TITLE
orange3 start pending on nltk_data Downloading

### ISSUE THREAD

**Issue by phiedulxp** [biolab/orange3#2548]
##### Orange version

orange3=3.4.5
python 3.6.2

##### Expected behavior

show orange gui

##### Actual behavior
pending on this:
[nltk_data] Downloading package wordnet to
[nltk_data]     C:\Users\lxp\AppData\Roaming\Orange\nltk_data...

##### Steps to reproduce the behavior
conda install -n orange orange3-text orange3-network
orange-canvas

##### Additional info (worksheets, data, screenshots, ...)
when I check my nltk download, it show that wordnet is up-to-date!
![image](https://user-images.githubusercontent.com/14953806/29714482-059248a4-89d6-11e7-9bb3-873ded81c99d.png)

**Comment by ajdapretnar** [comment]
Probably have to solve it in https://github.com/biolab/orange3-text/issues.
@nikicc

**Comment by phiedulxp** [comment]
```
[nltk_data] Downloading package wordnet to
[nltk_data]     C:\Users\xxx\AppData\Roaming\Orange\nltk_data...
[nltk_data]   Package wordnet is already up-to-date!
[nltk_data] Downloading package stopwords to
[nltk_data]     C:\Users\xxx\AppData\Roaming\Orange\nltk_data...
[nltk_data]   Package stopwords is already up-to-date!
[nltk_data] Downloading package punkt to
[nltk_data]     C:\Users\xxx\AppData\Roaming\Orange\nltk_data...
[nltk_data]   Package punkt is already up-to-date!
[nltk_data] Downloading package averaged_perceptron_tagger to
[nltk_data]     C:\Users\xxx\AppData\Roaming\Orange\nltk_data...
```
it work to show the start up unless open 3 orange-canvas process at the same time,

**Comment by phiedulxp** [comment]
start up with error :
```
4540 E:\Anaconda2\envs\orange\Library\bin\ssleay32.dll
```

**Comment by kernc** [comment]
Duplicate of https://github.com/biolab/orange3-text/issues/223.

> start up with error :

@phiedulxp If this is preventing Orange or Text add-on to start (after NLTK data had been successfully downloaded), please open an issue on [Orange3-Text](https://github.com/biolab/orange3-text/issues/223).
